src/Analyzer.Utilities/TemplateDiscovery.cs

```diff
@@ -9,16 +9,38 @@
 using Microsoft.Extensions.Logging;
 using Newtonsoft.Json;
 
-namespace Microsoft.Azure.Templates.Analyzer.Cli
+namespace Microsoft.Azure.Templates.Analyzer.Utilities
 {
     /// <summary>
     /// Holds a template file with a parameters file to use with analysis, if applicable.
     /// </summary>
-    /// <param name="Template">Template to be analyzed.</param>
-    /// <param name="Parameters">Parameters to use when analyzing the template.</param>
-    public record TemplateAndParams(FileInfo Template, FileInfo Parameters);
```

Comment:
I think we should just convert all the projects to net6.0 instead of netstandard.  There's no reason for the other projects to stay netstandard anymore.

Reply:
Sounds good to me, especially when net7.0 seems necessitated soon...

Reply:
With the update, this should be reverted back to a `record`.